frontend/config/webpack.config.js

```diff
@@ -4,21 +4,39 @@ const path = require('path');
 const webpack = require('webpack');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const CleanWebpackPlugin = require('clean-webpack-plugin');
+const InlineChunkWebpackPlugin = require('html-webpack-inline-chunk-plugin');
 
 const srcPath = path.join(__dirname, '..', 'scripts');
 
 const templates = require('./static.templates');
 
 module.exports = {
-  entry: path.join(srcPath, 'index'),
+  entry: {
+    main: path.join(srcPath, 'index')
+  },
   output: {
     filename: '[name].[chunkhash].js',
     chunkFilename: '[name].[chunkhash].js',
     path: path.resolve(__dirname, '..', 'dist'),
     publicPath: '/'
   },
   plugins: [
-    new webpack.optimize.CommonsChunkPlugin({ name: 'common' }),
+    new webpack.optimize.CommonsChunkPlugin({ name: 'common', minChunks: 3 }),
```

Comment:
This doesn't seem to be test related. could you explain, please?

Reply:
The test specific branch is #275 . This is related to performance improvements. Sets a minimum of 3 common chunks in order to extract a module into the common bundle.